# DoodleVPN owned-acquisition plan — 31 July 2026

## tl;dr

Existing acquaintance audiences are exhausted, so the previous partner-first recommendation is invalid. The selected motion is **Doodle Rescue**: a one-gigabyte monthly emergency tier for new users, the already-built DoodleRay Network Doctor, and problem-led distribution through search, short video, and genuine community answers. Referral rewards remain a secondary multiplier.

The score is a transparent decision prior, not a forecast. The economics show the free tier breaks even before support at roughly 4.0% 30-day paid conversion when using the conservative fully allocated server cost of $0.24 per activated user-month.

## Context & Methods

### Key Assumptions

- Internal aggregates cover business flows from 13 April through 30 July 2026.
- New non-referral payers produce about $6.03 of 30-day net revenue.
- Fully allocated infrastructure cost is about $0.24 per active user-month; marginal cost is unknown and likely lower.
- The free pilot is limited to genuinely new users from dedicated acquisition links, excluding active and prior payers to control cannibalization.
- Scores compare complete go-to-market plays against the owner's constraints; they do not claim causal lift.

In [1]:
import csv
from pathlib import Path

ROOT = Path('analysis/doodlevpn-owned-acquisition-plan-2026-07-31')

def read_csv(name):
    with (ROOT / name).open(encoding='utf-8') as handle:
        return list(csv.DictReader(handle))

plays = read_csv('play_scoring.csv')
economics = read_csv('rescue_economics.csv')

## Data

In [2]:
plays[:5]

[{'play': 'Doodle Rescue: 1 GB reserve + Network Doctor + problem capture', 'cold_pull': '95', 'doodle_fit': '95', 'zero_upfront': '90', 'speed_to_evidence': '75', 'scalability': '90', 'economics_measurement': '80', 'verdict': 'Primary 90-day engine', 'score': '89.0', 'rank': '1'}, {'play': 'Network Doctor + current 3-day trial', 'cold_pull': '75', 'doodle_fit': '95', 'zero_upfront': '95', 'speed_to_evidence': '90', 'scalability': '80', 'economics_measurement': '90', 'verdict': 'Fallback if free reserve fails', 'score': '86.5', 'rank': '2'}, {'play': 'Manual expert answers in communities', 'cold_pull': '72', 'doodle_fit': '85', 'zero_upfront': '95', 'speed_to_evidence': '80', 'scalability': '55', 'economics_measurement': '65', 'verdict': 'Distribution layer for Rescue', 'score': '76.0', 'rank': '3'}, {'play': 'Outsider micro-affiliates on verified revenue only', 'cold_pull': '68', 'doodle_fit': '65', 'zero_upfront': '90', 'speed_to_evidence': '70', 'scalability': '80', 'economics_measu

In [3]:
economics

[{'scenario': 'Kill boundary', 'activated_free_users': '100', 'paid_conversion': '0.03', 'new_payers': '3.0', 'revenue_30d': '18.09', 'server_cost_30d': '24.0', 'contribution_before_support': '-5.91'}, {'scenario': 'Base', 'activated_free_users': '100', 'paid_conversion': '0.06', 'new_payers': '6.0', 'revenue_30d': '36.18', 'server_cost_30d': '24.0', 'contribution_before_support': '12.18'}, {'scenario': 'Scale', 'activated_free_users': '100', 'paid_conversion': '0.08', 'new_payers': '8.0', 'revenue_30d': '48.24', 'server_cost_30d': '24.0', 'contribution_before_support': '24.24'}]

## Results

In [4]:
gap = {"may_revenue": 3581.06, "july_revenue": 2823.94, "monthly_gap": 757.12, "revenue_per_new_payer_30d": 6.03, "payers_to_close_gap": 125.6, "activated_free_at_6pct": 2093, "qualified_visitors_at_30pct_activation": 6977}
break_even = 0.24 / 6.03
assert plays[0]['play'].startswith('Doodle Rescue')
assert 0.039 < break_even < 0.041
{
    'winner': plays[0]['play'],
    'winner_score': float(plays[0]['score']),
    'break_even_paid_conversion': round(break_even, 4),
    'monthly_revenue_gap': gap['monthly_gap'],
    'payers_to_close_gap': gap['payers_to_close_gap'],
    'activated_free_users_needed_at_6pct': gap['activated_free_at_6pct'],
    'qualified_visitors_needed_at_30pct_activation': gap['qualified_visitors_at_30pct_activation'],
}

{'winner': 'Doodle Rescue: 1 GB reserve + Network Doctor + problem capture', 'winner_score': 89.0, 'break_even_paid_conversion': 0.0398, 'monthly_revenue_gap': 757.12, 'payers_to_close_gap': 125.6, 'activated_free_users_needed_at_6pct': 2093, 'qualified_visitors_needed_at_30pct_activation': 6977}

## Takeaways

1. The immediate objective is not vanity registrations but positive 30-day contribution per acquired cohort.
2. At the conservative cost assumption, 6% free-to-paid conversion yields about $12 contribution per 100 activated free users before support and development; 3% loses about $6.
3. Closing the entire May-to-July revenue gap requires roughly 126 additional payers per month, or about 2,100 activated free users at 6% conversion. This cannot be promised immediately without paid reach.
4. The first pilot should stop or change if 500 activated new free users produce below 3% paid conversion, server cost exceeds $0.35 per free active month, or fraud/support load breaks the model.
5. If the permanent reserve tier fails, keep the Network Doctor and problem-capture distribution but revert the offer to the current three-day trial.